docs/proposals/capacity-planning-refactoring.md

```diff
@@ -0,0 +1,100 @@
+# Proposal: Refactoring of capacity planning
+
+Last updated: 09.08.2021
+
+
+## Abstract
+
+Refactor logic of AC reservation to consider cases for 2 or more LVG PVs
```

Comment:
> Refactor logic of AC reservation to consider cases for 2 or more LVG PVs

This sounds confusing. Rename to something like `Need to improve disk space usage for LVG based persistent volumes`

Reply:
updated